test/glific_web/schema/ticket_test.exs

```diff
@@ -58,6 +58,18 @@ defmodule GlificWeb.Schema.TicketTest do
 
     tickets = get_in(query_data, [:data, "tickets"])
     assert length(tickets) == 1
+
+    result = auth_query_gql_by(:list, user, variables: %{"filter" => %{"name" => "Adelle Cavin"}})
+    assert {:ok, query_data} = result
+
+    tickets = get_in(query_data, [:data, "tickets"])
+    assert length(tickets) == 1
+
+    result = auth_query_gql_by(:list, user, variables: %{"filter" => %{"phone" => 704651301}})
+    assert {:ok, query_data} = result
+
+    tickets = get_in(query_data, [:data, "tickets"])
+    assert length(tickets) == 1
```

Comment:
filter name is **name_or_phone** and not **name** or **phone** separately.
Also check why this testcase is passing